ami/main/api/views.py

```
@@ -1608,17 +1624,111 @@ def list(self, request, *args, **kwargs):
 
-class TaxaListViewSet(viewsets.ModelViewSet, ProjectMixin):
+class TaxaListViewSet(DefaultViewSet, ProjectMixin):
     queryset = TaxaList.objects.all()
+    serializer_class = TaxaListSerializer
+    ordering_fields = [
+        "name",
+        "description",
+        "annotated_taxa_count",
+        "created_at",
+        "updated_at",
+    ]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)